tests/fixtures/test-graph-deployment/README.md

```diff
@@ -1,271 +1,20 @@
-
-## Structure
-
-```
-test-graph-deployment/
-├── langgraph.json          # LangGraph configuration
-├── requirements.txt        # Python dependencies
-├── .env.example           # Environment variable template
-├── README.md              # This file
-├── DEPLOYMENT_GUIDE.md    # Detailed deployment instructions
-└── test_agent/            # Python module
-    ├── __init__.py
-    └── agent.py           # Minimal echo graph implementation
-```
-
-## Graph Implementation
-
-The test graph is intentionally minimal:
-
-- **Single Node**: An `echo` node that prefixes messages with "Echo: "
-- **Simple State**: Contains only a `message` field
-- **No Dependencies**: Does not require API keys or external services
-- **Fast Execution**: Completes immediately for quick test cycles
-
-### Graph Flow
-
-```
-START → echo_node → END
-```
-
-**Input:**
-```json
-{
-  "message": "Hello, World!"
-}
-```
-
-**Output:**
-```json
-{
-  "message": "Echo: Hello, World!"
-}
-```
+> **📍 Testing standards centralized**
+>
+> For deployment fixture patterns and usage, see `@docs/dev/testing/test-fixtures.md`.
 
 ## Quick Start
 
-### 1. Review the Code
-
-The graph implementation is in `test_agent/agent.py`:
-
-```python
-from langgraph.graph import StateGraph, END
-
-# Define state
-class State(TypedDict):
-    message: str
-
-# Define node
-def echo_node(state: State) -> State:
-    return {"message": f"Echo: {state['message']}"}
-
-# Build graph
-builder = StateGraph(State)
-builder.add_node("echo", echo_node)
-builder.set_entry_point("echo")
-builder.add_edge("echo", END)
-
-# Compile
-graph = builder.compile()
-```
-
-### 2. (Optional) Test Locally
-
-If you want to test the graph locally before deploying:
-
-```bash
-# Install dependencies
-pip install -r requirements.txt
-
-# Install LangGraph CLI
-pip install langgraph-cli
-
-# Run local development server
-langgraph dev
-
-# In another terminal, test the graph
-curl -X POST http://localhost:8000/runs \
-  -H "Content-Type: application/json" \
-  -d '{"input": {"message": "test"}}'
-```
-
-### 3. Deploy to LangGraph Cloud
-
-See **[DEPLOYMENT_GUIDE.md](./DEPLOYMENT_GUIDE.md)** for detailed step-by-step instructions.
-
-**Summary:**
+This directory contains a minimal LangGraph echo application for integration testing.
 
-1. Navigate to [LangSmith](https://smith.langchain.com/)
-2. Go to "Deployments" section
-3. Click "New Deployment"
-4. Connect this repository or upload files
-5. Configure deployment settings
-6. Deploy
-7. Note the **Graph ID** from the deployment details
-
-### 4. Set Environment Variables
-
-After deployment, set these environment variables for integration tests:
+**Deployment:** See `DEPLOYMENT_GUIDE.md` for step-by-step deployment instructions.
 
+**Environment Variables:**
 ```bash
-# Required for all tests
 export LANGSMITH_API_KEY="<your-api-key>"
 export LANGSMITH_WORKSPACE_ID="<your-workspace-id>"
-
-# Required for assistant tests
 export TEST_GRAPH_ID="<graph-id-from-deployment>"
```

Comment:
remove; out of date

Reply:
Fixed in commit 94e7b90: Removed outdated TEST_GRAPH_ID environment variable from test-graph-deployment README.